specs/extension_image.md

```diff
@@ -35,8 +35,109 @@ confext images extend `/etc`. They must contain a `/etc/extension-release.d/exte
 to identify them.
 
 ## Image Content
-Extension Images should be additive, and not override content present in the base image or other DDIs,
-but this will not be enforced.
+Extension Images should be additive, and not override content present in the base image or other DDIs.
+However, there currently is no safe and efficient way to detect collisions and to enforce content uniqueness
+across the stack of images.
+In a future version of this specification options for enforcing uniqueness may be provided.
+
+## Base Directory Immutability / Mutability
+The contents of system and configuration extension images are immutable and cannot be changed after
+the image was created.
+By default, the base directory (`/etc`, `/usr`, `/opt`) overlaid by an extension is also rendered immutable
+when the extension is applied ("merged").
+This only affects base directories actually contained in extensions merged to a root filesystem:
+for instance, if a sysext extends `/usr` but not `/opt`, `/opt` will remain mutable.
+However, immutability affects the _full_ base directory.
+An extension shipping a single custom path e.g. below `/etc/appconfig-extra/some/sub/path` will still
+render the entirety of `/etc` immutable.
+The base directory or directories remain immutable for as long as the extensions are merged.
+Original state is regained only after all extensions overlaying a base directory were un-merged.
```

Comment:
s/were/have been/ ? I'm not a native speaker, but "were" sounds to me like it must be in the past, and this is potentially in the future.

Reply:
Yeah: "you can go out after you have done your homework", and not "you can go out after you did your homework".

Reply:
Will fix, happened in multiple places.